### PR TITLE
test: Make sure that components are honoring a correct default display value

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #2513

## Summary

test: Make sure that components are honoring a correct default display value.

### CODE REVIEW

File: src/components/calcite-radio-button/calcite-radio-button.tsx [hunk 1]
```diff
@@ -257,7 +255,7 @@ export class CalciteRadioButton {
   @Listen("click")
   check(event: MouseEvent | FocusEvent): void {
     // Prevent parent label from clicking the first radio when calcite-radio-button is clicked
-    if (this.el.closest("label") && (event.target === this.el || event.target === this.radio)) {
+    if (this.el.closest("label") && event.composedPath().includes(this.el)) {
```
Comment (jcfranco) [inline src/components/calcite-radio-button/calcite-radio-button.tsx:258]: This is unrelated. Please separate this out. This should be a `fix`.

**Comment by jcfranco** [comment]
@driskull Thanks for helping out with this! Can you extract out the radio-button fix before merging?

**Review by jcfranco — APPROVED** [review]
👍🏼

File: src/components/calcite-radio-button/calcite-radio-button.tsx (hunk 1 above)
Comment (driskull) [reply src/components/calcite-radio-button/calcite-radio-button.tsx:258]: @jcfranco I had to do this because the test was failing since the radio switched from inline to inline-block.

Comment (driskull) [reply src/components/calcite-radio-button/calcite-radio-button.tsx:258]: I can skip the test for now if that is what you would like.

Comment (driskull) [reply src/components/calcite-radio-button/calcite-radio-button.tsx:258]: These two tests are failing from the change from inline to inline-block.

https://github.com/Esri/calcite-components/blob/master/src/components/calcite-label/calcite-label.e2e.ts#L610-L664

Comment (jcfranco) [reply src/components/calcite-radio-button/calcite-radio-button.tsx:258]: If this was needed because of the display changes then it's fine. Ignore my comment.
